packages/date-picker/test/keyboard-input.test.js

```diff
@@ -678,13 +664,6 @@ describe('keyboard', () => {
       await sendKeys({ press: 'Enter' });
       expect(datepicker.value).to.equal('2000-01-01');
     });
-
-    it('should be invalid on enter with false input', async () => {
```

Comment:
This is essentially a duplicate of the following test:

https://github.com/vaadin/web-components/blob/59b9864003c9dec13d50c7687d48cde43daa32bd/packages/date-picker/test/validation.test.js#L217